sphinx/testing/path.py

```diff
@@ -98,6 +98,16 @@ def copytree(self, destination: str, symlinks: bool = False) -> None:
             pointed to by the symbolic links are copied.
         """
         shutil.copytree(self, destination, symlinks=symlinks)
+        # If source tree is marked read-only (e.g. because it is on a read-only
+        # filesystem), `shutil.copytree` will mark the destination as read-only
+        # as well.  To avoid failures when adding additional files/directories
+        # to the destination tree, ensure destination directories are not marked
+        # read-only.
+        if hasattr(os, 'chmod'):
```

Comment:
Why do you check `hasattr` here? Is there any environment not having `os.chmod`?

Reply:
I assumed Windows did not have it, but checking now I see that `os.chmod` is defined on Windows as well, so that check can be removed.